api/types/Message.js

```diff
@@ -58,6 +58,7 @@ const Message = /* GraphQL */ `
 
   extend type Mutation {
     addMessage(message: MessageInput!): Message
+      @rateLimit(max: 20, window: "1m")
```

Comment:
This might actually be too low. For a fast conversation, wdyt? Maybe it should be 30?

Reply:
30 per minute? Okay

Reply:
I don't know...i mean 20 seems high, I'm just imaging a text thread where things are going fast. 20 is probably reasonable though.